core/style.css

```diff
@@ -128,6 +128,22 @@ a:hover {
   text-decoration: underline;
 }
 
+.leaflet-control {
+  user-select: none; 
+}
+
+.leaflet-control-container .leaflet-top.leaflet-left {
+  width: 700px;  /* chat default width */
+  height: calc(100% - 126px); /* chat default start */
+  max-height: 90%;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  align-items: flex-start;
+}
```

Comment:
Can we keep just those lines?
It seems like a good idea to me to leave `#portal_highlight_select` in `.leaflet-top.leaflet-left` so that all the controls are in one place. Otherwise it becomes necessary to manually fix their positioning in different parts of the code.

Reply:
When the highlight control is inside the same container as all buttons it will define the column width.
The second column would start on the right of it instead of a compact block below it.

Reply:
![grafik](https://github.com/user-attachments/assets/69461f67-1ae6-4d46-8c17-d802185d749c)